### PR TITLE
Addition of RaML (Resource Aware ML)

### PR DESCRIPTION
**Pull request by npereira97**

### CODE REVIEW

File: learn-ocaml-client.install [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "_build/install/default/lib/learn-ocaml-client/META" {"META"}
+  "_build/install/default/lib/learn-ocaml-client/opam" {"opam"}
+]
+bin: [
+  "_build/install/default/bin/learn-ocaml-client" {"learn-ocaml-client"}
+]
+doc: [
+  "_build/install/default/doc/learn-ocaml-client/LICENSE"
+  "_build/install/default/doc/learn-ocaml-client/README.md"
+]
```
Comment (hannelita) [inline learn-ocaml-client.install]: We should not commit this file

File: raml/ocaml-tree.patch [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+-    val set : t -> int -> char -> unit
+-    val blit : t -> int -> t -> int -> int -> unit
++    val set : bytes array -> int -> char -> unit
++    val blit : string array -> int -> bytes array -> int -> int -> unit		
+     val output : out_channel -> t -> int -> int -> unit
+-    val unsafe_blit_to_string : t -> int -> string -> int -> int -> unit
++    val unsafe_blit_to_string : string array -> int -> bytes -> int -> int -> unit
+     val input_bytes : in_channel -> int -> t
+   end
+
```
Comment (hannelita) [inline raml/ocaml-tree.patch:385]: What is this file? Do we need it?

Comment (npereira97) [reply raml/ocaml-tree.patch:385]: I'm not really sure what it does exactly. It is needed to build RaML and came along with the included files.

File: learn-ocaml-client.install (hunk 1 above)
Comment (npereira97) [reply learn-ocaml-client.install]: I've deleted it

**Comment by hannelita** [comment]
Note: I'll send another PR later on making a few adjustments on the docker files and replacing `ENV PATH $PATH:/home/opam/.opam/4.06/bin`with `ENV PATH $PATH:/home/opam/.opam/4.05/bin` [apparently there was a conflict ]

**Pull request merged**